src/components/InternalDropdown/helpers/getAllValues.js

```diff
@@ -0,0 +1,19 @@
+import normalizeValue from './normalizeValue';
+
+export default function getAllValues(children) {
+    if (!Array.isArray(children)) {
+        return [];
+    }
+    const values = [];
+    children.forEach(child => {
+        const { label: childLabel, name, value: childValue, variant: childVariant } = child;
+        if (childVariant === 'default') {
+            values.push({
+                label: childLabel,
+                name,
+                value: childValue,
+            });
+        }
+    });
```

Comment:
here we should take a more functional approach like using map or reduce

Reply:
Done